CONTRIBUTING.md

```diff
@@ -11,7 +11,110 @@ Thank you for your interest in contributing to CUDA Python! Based on the type of
+│                                    BUILD STAGE                                     │
+│                                                                                     │
+│  ┌─────────────────┐  ┌─────────────────┐  ┌─────────────────┐                     │
+│  │   linux-64      │  │ linux-aarch64   │  │     win-64      │                     │
+│  │  Self-hosted    │  │  Self-hosted    │  │ GitHub-hosted   │                     │
+│  │ linux-amd64-cpu8│  │ linux-arm64-cpu8│  │  windows-2022   │                     │
+│  └─────────────────┘  └─────────────────┘  └─────────────────┘                     │
+│                                                                                     │
+│  • Python versions: 3.9, 3.10, 3.11, 3.12, 3.13                                  │
+│  • CUDA version: 13.0.0 (build-time)                                              │
+│  • Components: cuda-core, cuda-bindings, cuda-pathfinder, cuda-python             │
+└─────────────────────────────────────────────────────────────────────────────────────┘
+                                             │
+                                             ▼
+┌─────────────────────────────────────────────────────────────────────────────────────┐
+│                               ARTIFACT STORAGE                                     │
+│                                                                                     │
+│  ┌──────────────────────────┐              ┌──────────────────────────┐            │
+│  │     GitHub Artifacts     │              │      GitHub Cache        │            │
+│  │   • Wheel files (.whl)   │              │ • Build dependencies     │            │
+│  │   • Test artifacts       │              │ • Compiled binaries      │            │
+│  │   • Documentation        │              │ • Python environments    │            │
+│  │   (30-day retention)     │              │ • Package caches         │            │
+│  └──────────────────────────┘              └──────────────────────────┘            │
+└─────────────────────────────────────────────────────────────────────────────────────┘
+                                             │
+                                             ▼
+┌─────────────────────────────────────────────────────────────────────────────────────┐
+│                                    TEST STAGE                                      │
+│                                                                                     │
+│  ┌─────────────────┐  ┌─────────────────┐  ┌─────────────────┐                     │
+│  │   linux-64      │  │ linux-aarch64   │  │     win-64      │                     │
+│  │  Self-hosted    │  │  Self-hosted    │  │ GitHub-hosted   │                     │
+│  │   GPU runners   │  │   GPU runners   │  │   GPU runners   │                     │
+│  │   (L4/A100)     │  │     (A100)      │  │                 │                     │
```

Comment:
ditt, GPU models (or the fact that all test runners have GPUs) do not matter, remove them